frontend/src/components/PrivateRoute/index.jsx

```diff
@@ -1,25 +1,29 @@
 import React from "react";
 import { Route, Redirect } from "react-router-dom";
 import Loader from "../Loader";
+import Navbar from "../Navbar";
 
 function PrivateRoute({ component: Component, authed, ...rest }) {
   return (
-    <Route
-      {...rest}
-      render={props => {
-        if (authed === true) {
-          return <Component {...props} />;
-        }
-        if (authed === false) {
-          return (
-            <Redirect
-              to={{ pathname: "/login", state: { from: props.location } }}
-            />
-          );
-        }
-        return <Loader></Loader>;
-      }}
-    />
+    <div>
+      <Navbar />
+      <Route
+        {...rest}
+        render={props => {
+          if (authed === true) {
```

Comment:
small nit: you can just write `if (authed) {...}`. Similarly on line 16, `if (!authed) { ... }`